### PR TITLE
fix: Typos in GitHub actions declarations

### PR DESCRIPTION
**Pull request by sawood14012**

### CODE REVIEW

File: .github/workflows/ci-build-image.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
         with:
           issue-number: ${{ github.event.client_payload.github.payload.issue.number }}
           body: |
-            :test_tube: [CI has Started Image Build]( https://github.com/${{github.repository}}/actions/runs/${{github.run_id}})" }'
+            :test_tube: [CI has Started Image Build]( https://github.com/${{github.repository}}/actions/runs/${{github.run_id}})
```
Comment (arajkumar) [inline .github/workflows/ci-build-image.yml:26]: Do we need an another comment just to show it is building? Why not add an emoji to indicate the progress and done, then finally a comment to show the full url of the image?

**Review by arajkumar — CHANGES_REQUESTED** [review]
Please fix the created docker image tag.

Instead of just PR number, please use the existing tag format.(e.g SNAPSHOT-PR-250)

File: .github/workflows/ci-build-image.yml (hunk 1 above)
Comment (sawood14012) [reply .github/workflows/ci-build-image.yml:26]: it already adds an emoji shall i remove this comment ?

**Comment by sawood14012** [comment]
> Please fix the created docker image tag.
> 
> Instead of just PR number, please use the existing tag format.(e.g SNAPSHOT-PR-250)

Done

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**